packages/bpk-component-calendar/src/bpk-calendar-nav.scss

```diff
@@ -47,22 +47,22 @@
     cursor: pointer;
     appearance: none;
 
+    @include bpk-themeable-property(color, --bpk-calendar-nudger-icon-color, $bpk-calendar-nav-icon-fill);
+
     @include bpk-hover {
-      .bpk-calendar-nav__icon {
-        fill: $bpk-calendar-nav-icon-hover-fill;
-      }
+      @include bpk-themeable-property(color, --bpk-calendar-nudger-icon-hover-color, $bpk-calendar-nav-icon-hover-fill);
```

Comment:
Not sure I understand here, you're not applying the fill to the icon but to the hover of the button?

Reply:
I'm using `fill: currentColor` which means the icon picks it up from the button's color. Then I'm just setting the button color for the different states.